### ISSUE TITLE
FNX-5325 ⁃ [Bug] Custom tabs toolbar title is cut off with increased font size

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/fenix#1905]
## Steps to reproduce
* Set font size (Settings ~~> Accessibilty~~ > Font Size) to Large
* Open custom tab

### Expected behavior
Title is not cut off

### Actual behavior
Title is cut off

### Device information
* Android device: Pixel 2
* Fenix version: 1.0.1917

I wonder if we can get data on how many users change their font size to know how important it is to address issues like these: in the past (Fennec), we assumed it was an uncommon use case but devices are decreasing their font size (despite the fact that eyes are not getting upgraded) so it may be more common.

In the worst case, we can probably collect our own telemetry events for "is using standard font size" by comparing dp to sp (e.g. by converting pixels to dp and pixels to sp in code).

**Comment by vesta0** [comment]
Opened #2265 to track all font related issues.

**Comment by vesta0** [comment]
@eeejay we currently have 3 bugs related to larger font size (will be tagging you in all of them). Let us know what your recommended approach would be.

**Comment by vesta0** [comment]
@eeejay see ^^ :)

**Comment by eeejay** [comment]
Thanks, I should have replied earlier. I don't have much insight on this, I think these are UX questions as to how text should scale in these places to give the most useful information.

In general I would seek parity with other similar components in apps like Chrome.

**Comment by sblatz** [comment]
I have a pending patch in a-c: https://github.com/mozilla-mobile/android-components/pull/3214

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest build 1.0.1923(#11560607), with Huawei P9 Lite (Android 6), and Xiaomi Mi8 Lite (Android 8.1).

**Comment by sblatz** [comment]
@vesta0 per our meeting, this issue will be coming back when I revert my a-c change.

**Comment by sblatz** [comment]
This is blocked on https://github.com/mozilla-mobile/android-components/issues/3213

**Comment by NotWoods** [comment]
@sblatz Is this still blocked/still an issue?

**Comment by sblatz** [comment]
No, this seems to be working properly. Let's have QA verify though 😄

**Comment by sv-ohorvath** [comment]
Verified as fixed on Nightly 9/27.
Samsung Galaxy Tab 3 (Android 8.0), Samsung Galaxy S8 (Android 9)
